### DISCUSSION TITLE
Fabric compatibility with Mistral 7B

### DISCUSSION THREAD

**Discussion by TestudoSpatialis** [danielmiessler/Fabric#974] [General]
Hi All,

Unfortunately, I have to deal with a macbook air M2 8GB and I know I'm pushing it to its limits. The feature I am fond of the most is extract_wisdom. However, using this only gives me a basic summary of three paragraphs, so it seems like Mistral does not take into account all the output instructions that are given in the pattern's system.md. 

At this moment I'm copying the contents of the system.md file and paste it into Google Gemini together with the subtitles of the youtube video that I download with yt-dlp haha. That works, but naturally I want to do things from the terminal.  I have a couple of questions:

- What other features of fabric are compatible with smaller models like mistral 7B? Or did I really go out of my depth here?
- Does anyone use a payed API such as chatGPT for this? Which one is the best and how expensive is it?
- Will it work better if I quantize an LLM such as LLama 3.1? (LLama 3.1 7B makes my cpu go nuts)

Love to hear your thoughts.

Cheers,
The space turtle

**Comment by BryanDollery** [comment]
I know it's now somewhat later than when you originally asked, but I'm using ollama/mistral-7b locally on my laptop (9th gen 6-core i7, 40gb ram, gtx970 GPU with 4gb ram! - not exactly a powerhouse - running arch). And I use openai/chatgpt-4o. The local model is amazing, for what it is, but it's nowhere near as clever as 4o. I have a pipeline that automatically archives my questions and answers, so I use the local model for generating better queries, titles, tags, summaries, etc. but I get the main answer itself from gpt-4o. This is quite fast and efficient on my machine. The local model is slow, but overall you get a response faster than using a hosted model because of the latency inherent in these services.
